src/openms/source/METADATA/MetaInfoInterface.cpp

```diff
@@ -53,7 +54,8 @@ namespace OpenMS
     }
   }
 
-  MetaInfoInterface::MetaInfoInterface(MetaInfoInterface&& rhs) :
+  /// Move constructor
+  MetaInfoInterface::MetaInfoInterface(MetaInfoInterface&& rhs) noexcept :
     meta_(rhs.meta_)
```

Comment:
should be `std::move(rhs.meta_)`, just to make sure its still correct when someone changes the type of meta_ at some point.